contentlayer.config.ts

```diff
@@ -65,6 +65,7 @@ const includeDirs = [
   'docs/sway-standards/docs/src',
   'docs/sway-by-example-lib/docs/src',
   'docs/migrations-and-disclosures/docs/src',
+  'docs/integration-docs/docs',
```

Comment:
https://docs-csdbxly6o-fuel-labs.vercel.app/docs/nightly/integration-docs/

Integrations book on the nightly side does not exist

Reply:
Also need to include it in the directory Exclusions (excludeDirs)